pages/history.tsx

```diff
@@ -103,8 +125,25 @@ const History = () => {
         ),
       ]
         .concat(
-          transfers.map(
-            x => [-1, { ui: { timestamp: x.timestamp }, ...x }] as any
+          transfers[0].map(
+            x =>
+              [
+                TransferType.MUTEZ,
+                { ui: { timestamp: x.timestamp }, ...x },
+              ] as any
+          )
+        )
+        .concat(
+          transfers[1].map(
+            x =>
+              [
+                x.token.standard === "fa2"
+                  ? TransferType.FA2
+                  : x.token.standard === "fa1.2"
+                  ? TransferType.FA1_2
+                  : TransferType.UNKNOWN,
+                { ui: { timestamp: x.timestamp }, ...x },
+              ] as any
```

Comment:
I don't like this `any` but it looks like it has to change a lot..
so ... ok to leave it here.